#type_of_cam:
#0---stereo kitti
#1---stereo euroc
#2---T265

type_of_cam: 2

image_width: 848
image_height: 800

image_width_out: 848
image_height_out: 800

# image_width_out: 300
# image_height_out: 300

cam0_intrinsics: [286.368408203125, 286.3091125488281, 425.2561950683594, 403.61749267578125] #fx fy cx cy
cam0_distortion_coeffs: [-0.007536786142736673, 0.04236653074622154, -0.03889356181025505, 0.006278385873883963] #k1 k2 r1 r2
cam1_intrinsics: [285.8463134765625, 285.8080139160156, 416.856201171875, 403.4840087890625] #fx fy cx cy
cam1_distortion_coeffs: [-0.01030058041214943, 0.05014048144221306, -0.04687238112092018, 0.009046079590916634] #k1 k2 r1 r2

T_init:
[0, 0, 1, 0,
 -1, 0, 0, 0,
 0, -1, 0, 0,
 0, 0, 0, 1]

T_world_body:
[1,0,0,0,
 0,1,0,0,
 0,0,1,0,
 0,0,0,1]

T_body_cam0:
[0, 0, 1, 0,
-1, 0, 0, 0,
 0, -1, 0, 0,
 0, 0, 0, 1]

T_cam1_cam0:
[9.99994218e-01, -8.26635107e-04, -3.29655036e-03, -6.42428249e-02,
 8.28813005e-04,  9.99999523e-01,  6.59199432e-04, -1.58874827e-05,
 3.29600368e-03, -6.61927043e-04,  9.99994397e-01, 3.78768891e-06,
 0, 0, 0, 1]

#reserved para for future functions
run_rviz: true
